packages/bpk-svgs/src/icons/icons.test.js

```diff
@@ -0,0 +1,14 @@
+import fs from 'fs';
+
+// In order for the docs site to serve a zip-file containing all icon SVGs, the `zip-it-loader` loader
+// must be provided with an empty file within the icons directory.
+// If icons.js is not empty, then it will be served instead of the SVGs
+describe('icons.js', () => {
+  it('icons.js file should be empty', () => {
+    const iconsJsFileContents = fs.readFileSync(
+      'packages/bpk-svgs/src/icons/icons.js',
```

Comment:
shouldn't this path just be `./icons.js` if the test file is in the same directory? 🤔

Reply:
Yeah it should. I just blindly copied your code because I'm stupid 🤦 Will update.